hazelcast/src/main/java/com/hazelcast/internal/metrics/MetricDescriptorConstants.java

```diff
@@ -0,0 +1,584 @@
+    public static final String CLIENT_METRIC_CONNECTION_CONNECTIONID = "connectionId";
+    public static final String CLIENT_METRIC_CONNECTION_EVENT_HANDLER_COUNT = "eventHandlerCount";
+    public static final String CLIENT_METRIC_CONNECTION_CLOSED_TIME = "closedTime";
+    public static final String CLIENT_METRIC_INVOCATIONS_PENDING_CALLS = "pendingCalls";
+    public static final String CLIENT_METRIC_INVOCATIONS_STARTED_INVOCATIONS = "startedInvocations";
+    public static final String CLIENT_METRIC_INVOCATIONS_MAX_CURRENT_INVOCATIONS = "maxCurrentInvocations";
+    public static final String CLIENT_METRIC_LISTENER_SERVICE_EVENT_QUEUE_SIZE = "eventQueueSize";
+    public static final String CLIENT_METRIC_LISTENER_SERVICE_EVENTS_PROCESSED = "eventsProcessed";
+    // ===[CLIENT]======================================================
+
+    // ===[CLUSTER]=====================================================
+    public static final String CLUSTER_PREFIX = "cluster";
+    public static final String CLUSTER_PREFIX_CLOCK = "cluster.clock";
+    public static final String CLUSTER_PREFIX_CONNECTION = "cluster.connection";
+    public static final String CLUSTER_PREFIX_HEARTBEAT = "cluster.heartbeat";
+    public static final String CLUSTER_DISCRIMINATOR_ENDPOINT = "endpoint";
+    public static final String CLUSTER_METRIC_CLUSTER_CLOCK_MAX_CLUSTER_TIME_DIFF = "maxClusterTimeDiff";
+    public static final String CLUSTER_METRIC_CLUSTER_CLOCK_CLUSTER_TIME = "clusterTime";
+    public static final String CLUSTER_METRIC_CLUSTER_CLOCK_CLUSTER_TIME_DIFF = "clusterTimeDiff";
+    public static final String CLUSTER_METRIC_CLUSTER_CLOCK_CLUSTER_UP_TIME = "clusterUpTime";
+    public static final String CLUSTER_METRIC_CLUSTER_CLOCK_LOCAL_CLOCK_TIME = "localClockTime";
+    public static final String CLUSTER_METRIC_CLUSTER_CLOCK_CLUSTER_START_TIME = "clusterStartTime";
+    public static final String CLUSTER_METRIC_HEARTBEAT_MANAGER_LAST_HEARTBEAT = "lastHeartbeat";
+    public static final String CLUSTER_METRIC_CLUSTER_SERVICE_SIZE = "size";
+    // ===[/CLUSTER]====================================================
+
+    // ===[CP SUBSYSTEM]================================================
+    public static final String CPSUBSYSTEM_PREFIX_RAFT = "raft";
+    public static final String CPSUBSYSTEM_PREFIX_RAFT_GROUP = "raft.group";
+    public static final String CPSUBSYSTEM_PREFIX_RAFT_METADATA = "raft.metadata";
+    public static final String CPSUBSYSTEM_DISCRIMINATOR_GROUPID = "groupId";
+    public static final String CPSUBSYSTEM_TAG_NAME = "name";
+    public static final String CPSUBSYSTEM_METRIC_METADATA_RAFT_GROUP_MANAGER_GROUPS = "groups";
+    public static final String CPSUBSYSTEM_METRIC_METADATA_RAFT_GROUP_MANAGER_ACTIVE_MEMBERS = "activeMembers";
+    public static final String CPSUBSYSTEM_METRIC_METADATA_RAFT_GROUP_MANAGER_ACTIVE_MEMBERS_COMMIT_INDEX =
```

Comment:
Can we shorten the prefix here? The constant name is barely readable, at least for me.

Reply:
It became `CP`.